scripts/expand_tags.py

```diff
@@ -18,7 +18,11 @@ def main():
                               "be seperated by commas without spaces.\n"))
     args = parser.parse_args()
 
-    m = Mesh(structured=True, mesh=args.mesh_file)
+    try:
+        m = Mesh(structured=True, mesh=args.mesh_file)
+    except:
```

Comment:
Exceptions should only catch a the type of error that is expected (eg KeyError). Please qualify this. Thanks!